src/core/actions/index.js

```diff
@@ -1,18 +1,23 @@
 import {
+  LOG_OUT_USER,
   SET_JWT,
   SET_CLIENT_APP,
   SET_LANG,
   ENTITIES_LOADED,
   SET_CURRENT_USER,
 } from 'core/constants';
 
-export function setJWT(token) {
+export function setJwt(token) {
```

Comment:
Yeah, I think we've been formatting it like this so things like `setJwtToken` are a bit clearer. I think that looks better than `setJWTToken`.